recipes/rmm/meta.yaml

```diff
@@ -0,0 +1,51 @@
+{% set name = "rmm" %}
+{% set version = "21.12.00" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/rapidsai/rmm/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 71802422a04f2e0a828f7b7ec77e5cd1d24361d6dc5bf60a534659e4c2e08a44
+  folder: rmm
+
+build:
+  number: 0
+  skip: true  # [(not linux) or (cuda_compiler_version == "None") or (aarch64) or (py < 37) or (py >= 39)]
```

Comment:
1. Why would we contain the Python versions here? Conda-Forge currently supports PY37-PY310.
2. We need to build using `cuda_compiler_version == "11.2"` so that the `cudaMallocAsync` support can be included (cc: @jakirkham).

Reply:
@leofang 
The python dependency has been mentioned in https://github.com/rapidsai/rmm/blob/v21.12.00/conda/environments/rmm_dev_cuda11.0.yml.
Rather all package dependencies were listed from here only. 
Please let me know still we need to enable for py >=39.

Reply:
@leofang 
Also rest all changes has been done as per suggestion. Thanks for your help. Please have a look.

Reply:
See my suggestion below.

Reply:
Done all suggested changes.